Formula/jhipster.rb

```diff
@@ -0,0 +1,21 @@
+require "language/node"
+
+class Jhipster < Formula
+  desc "Generate, develop and deploy Spring Boot + Angular applications"
+  homepage "https://jhipster.github.io/"
+  url "https://registry.npmjs.org/generator-jhipster/-/generator-jhipster-4.5.2.tgz"
+  sha256 "5df0edbbdb685df5ae598b53fb2def43b6c23bb75d44763a63cda69567395954"
+
+  bottle :unneeded
```

Comment:
A bottle will be needed since it's building with npm, so you can just remove this.